testgrid/config.yaml

```diff
@@ -7557,8 +7557,8 @@ dashboards:
 # sig-windows dashboard
 - name: sig-windows
   dashboard_tab:
-  - name: "Conformance acs-engine on Azure"
-    description: Runs conformance tests on a Kubernetes cluster provided by acs-engine (https://github.com/Azure/acs-engine) on Azure cloud"
+  - name: "SIG-Windows Release (Azure)"
```

Comment:
I won't block on the name and am in favor of renaming to indicate it's about more than conformance at this point.  But a tiny style nit, I personally prefer short-all-lower names (https://testgrid.k8s.io/sig-release-master-blocking).  I would consider something like `azure-windows-master` to keep it short or `aks-engine-azure-windows-master` if you want to spell things out a bit more.

I nitted on the naming of a pull job based on aks-engine in https://github.com/kubernetes/test-infra/pull/11017#discussion_r252119501, I wonder if we could converge on something similar here.  The pattern I think we are close to following in most places is `{type}-kubernetes-e2e-{provisioner}-{cloud}[-{branch}][-{suffix}]`, eg: 
- `pull-kubernetes-e2e-kops-aws`
- `ci-kubernetes-e2e-kubeadm-gce-master`
- `ci-kubernetes-e2e-gce-cos-k8sstable1-serial` (where `k8sstable1` is a floating alias for latest release branch - 1, currently 1.13 since 1.14 was cut)

So in this case consider changing your test_group_name / the bucket you store results from `ci-kubernetes-e2e-win-1-13` to `ci-kuberentes-e2e-aks-engine-azure-master-windows` if you're running off of master

Reply:
+1 for `short-all-lower` names.